.env.example

```diff
@@ -5,3 +5,4 @@ REACT_APP_MAX_SELECTED_BERTHS=10
 REACT_APP_PIWIK_URL=https://your.piwik-url.com
 REACT_APP_PIWIK_ID=piwik_id
 REACT_APP_SENTRY_DSN=https://id1@sentry.io/id2
+REACT_APP_GRAPHQL_URI=https://venepaikka-api-gql.test.hel.ninja/graphql/
```

Comment:
weeeellll, we are gonna destroy this instance after the merge. maybe `http://localhost:8000/graphql/`?

Reply:
or something more examplary. not a real endpoint

Reply:
Oh, I see!
In this case, we have to implement a mock server for doing the job.

Reply:
I just meant that I would prefer not to put even our test endpoint here. But I don't know, maybe there is no harm if people know its URL.